sdcm/cluster.py

```diff
@@ -5011,35 +5027,24 @@ def node_setup(self, node, verbose=False, db_node_address=None, **kwargs):  # py
         scylla_repo_loader = self.params.get('scylla_repo_loader')
```

Comment:
not sure how critical this feature for loader,
as you've seen here this code is rarely used, and I'm not sure we'll care much about specific versions when building loader images (I do hope at some point we won't need those images, and docker base loaders would be used for everything)